src/arm/BBORG_COMMS-00A2.dts

```diff
@@ -14,94 +15,82 @@
 #include <dt-bindings/pinctrl/am33xx.h>
 #include <dt-bindings/gpio/gpio.h>
 
-/ {
-	compatible = "ti,beaglebone", "ti,beaglebone-black", "ti,beaglebone-green";
+/*
+* Helper to show loaded overlays under: /proc/device-tree/chosen/overlays/
+*/
+&{/chosen} {
+	overlays {
+		BBORG_COMMS-00A2 = __TIMESTAMP__;
+	};
+};
 
-	/* identification */
-	part-number = "BBORG_COMMS";
-	version = "00A2";
+/*
+	* Free up the pins used by the cape from the pinmux helpers.
+	*/
+&ocp {
+	P9_24_pinmux { status = "disabled"; };	/* P9_24: uart1_txd.d_can1_rx */
+	P9_26_pinmux { status = "disabled"; };	/* P9_26: uart1_rxd.d_can1_tx */
+	P9_13_pinmux { status = "disabled"; };	/* P9_13: gpmc_wpn.uart4_txd_mux2 */
+	P9_11_pinmux { status = "disabled"; };	/* P9_11: gpmc_wait0.uart4_rxd_mux2 */
+	P9_15_pinmux { status = "disabled"; };	/* P9_15: gpmc_a0.gpio1_16 */
+	P9_23_pinmux { status = "disabled"; };	/* P9_23: gpmc_a1.gpio1_17 */
+};
 
-	/* state the resources this cape uses */
-	exclusive-use =
-		/* the pin header uses */
-		//"P9.15", /* SINK A */
-		//"P9.23", /* SINK B */
-		//"P9.36", /* LOOP A */
-		//"P9.35", /* LOOP B */
-		"P9.24",	/* can1_rx */
-		"P9.26",	/* can1_tx */
-		"P9.13",		// uart4_txd
-		"P9.11",		// uart4_rxd
+&am33xx_pinmux {
+	bborg_comms_can_pins: pinmux_comms_can_pins {
+		pinctrl-single,pins = <
+			BONE_P9_24 (PIN_INPUT_PULLUP | MUX_MODE2)	/* P9_24: uart1_txd.d_can1_rx */
+			BONE_P9_26 (PIN_OUTPUT_PULLUP | MUX_MODE2)	/* P9_26: uart1_rxd.d_can1_tx */
+		>;
+	};
+	bborg_comms_rs485_pins: pinmux_comms_rs485_pins {
+		pinctrl-single,pins = <
+			BONE_P9_13 (PIN_OUTPUT | MUX_MODE6)	/* P9_13: gpmc_wpn.uart4_txd_mux2 */
+			BONE_P9_11 (PIN_INPUT  | MUX_MODE6)	/* P9_11: gpmc_wait0.uart4_rxd_mux2 */
+		>;
+	};
+	bborg_comms_sink_pins: pinmux_bborg_comms_sink_pins{
+		pinctrl-single,pins = <
+			BONE_P9_15 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)	/* P9_15: gpmc_a0.gpio1_16 */
+			BONE_P9_23 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)	/* P9_23: gpmc_a1.gpio1_17 */
+		>;
+	};
+};
 
-		/* the hardware ip uses */
-		"uart4",
-		"dcan1";
+&dcan1 {
+	status = "okay";
+	pinctrl-names = "default";
+	pinctrl-0 = <&bborg_comms_can_pins>;
+};
+	
+&uart4 {
+	status = "okay";
+	pinctrl-names = "default";
+	pinctrl-0 = <&bborg_comms_rs485_pins>;
+	//rs485-rts-delay = <0 0>;
```

Comment:
Why don't these need to be enabled? When should they?

Reply:
I guess when you need Request To Send (RTS) signal.